lib/core/DEVELOPMENT.md

```diff
@@ -0,0 +1,25 @@
+# Development guide - Core crate
+
+## Prerequisites
+To compile the Core crate you will first need to install:
+- [Protobuf](https://protobuf.dev/installation/)
+- [Emscripten](https://emscripten.org/docs/getting_started/downloads.html) for compiling to Wasm
```

Comment:
Should Firefox be mentioned? Isn't a compile prerequisite, but it's a test one

Reply:
Added [7d431ed](https://github.com/breez/breez-sdk-liquid/pull/809/commits/7d431ed82efd9c4a8877bf324164258818f8c3c2)